includes/classes/admin/mapping/field-types/acf.php

```diff
@@ -0,0 +1,96 @@
+<?php
+namespace GatherContent\Importer\Admin\Mapping\Field_Types;
+
+use GatherContent\Importer\Views\View;
+use WP_Query;
+
+class ACF extends Base implements Type {
+
+
+    protected $type_id = 'wp-type-acf';
+
+    /**
+     * Array of supported template field types.
+     *
+     * @var array
+     */
+    protected $supported_types = array(
+        'component',
+        'repeater',
+        'text_rich',
+    );
+
+    /**
+     * Creates an instance of this class.
+     *
+     * @since 3.0.0
+     */
+    public function __construct() {
+        $this->option_label = __( 'ACF Field Groups', 'gathercontent-import' );
+    }
+
+    public function underscore_template ( View $view ) {
+        global $wpdb;
+        global $wp_query;
+
+        // VARIABLES
+        $options_acf_groups = array();
+        $options_acf_groups_fields = array();
+
+        // ============= BUILD FIELD GROUP OPTIONS =============
+        $groups_query = "SELECT * FROM wp_posts WHERE post_type = 'acf-field-group' AND post_status = 'publish' AND post_parent = 0";
+        $group_results = $wpdb->get_results($groups_query);
+        
+        // FIELD GROUPS
+        if($group_results){
+            foreach($group_results as $group) {
+                // Set the top level field group array of options
+                $options_acf_groups[$group->post_name] = $group->post_title;
+                // Create a blank array based on the group id to define the groups fields
+                $options_acf_groups_fields[$group->post_name] = array();
+
+                // Define fields query
+                $fields_query = "SELECT * FROM wp_posts WHERE post_type = 'acf-field' AND post_content LIKE '%repeater%' AND post_parent = '$group->ID'"; // Use to get ONLY repeaters
```

Comment:
This is a slight N+1 query issue. Are we able to do a `whereIn` before the `foreach` for all the `$group->ID` and then pull them out as required. Also fixing the potentially slight security with just using the group id straight out of the DB.

E.g.

```php
$groupIds = array_map(function ($group) {
    return mysql_real_escape_string($group->ID);
}, $group_results);
$groupIds = implode(',', $groupIds);

$fields_query = "SELECT * FROM wp_posts WHERE post_type = 'acf-field' AND post_content LIKE '%repeater%' AND post_parent IN ($groupIds)";
$fields_results = $wpdb->get_results($fields_query);
$grouped_field_results = array_reduce($field_results, function ($carry, $field) {
    $carry[$field['post_parent']][] = $field;
    return $carry;
}, []);
```

Then you can pull the results when you foreach by doing

```php
$field_results = $grouped_field_results[$group->ID];
```